_posts/2017-12-19-2017-year-in-review.md

```diff
@@ -0,0 +1,144 @@
+
+## Flink Forward 2017
+
+2017 was the first year we ran a Flink Forward conference in both [Berlin](https://berlin-2017.flink-forward.org) (September 11-13) and [San Francisco](https://sf-2017.flink-forward.org) (April 10-11), and over 350 members of our community attended each event for speaker sessions, training, and discussion about Flink.
+
+[Slides](https://www.slideshare.net/FlinkForward/presentations) and [videos](https://www.youtube.com/channel/UCY8_lgiZLZErZPF47a2hXMA) are available for all speaker sessions, and if you're interested in learning more about how organizations use Flink in production we encourage you to browse and watch a couple.
+
+For 2018, Flink Forward will be back in [September in Berlin](https://flink-forward.org/), and in [April in San Francisco](https://sf-2018.flink-forward.org/).
+
+![Flink Forward Speakers]({{ site.baseurl }}/img/blog/speaker-logos-ff2017.png)
+
+## Features and Ecosystem
+
+### Stateful Stream Processing with dA Platform 2
+
+In September dataArtisans announced an early access program of our own SaaS product based on Apache Flink, [dA Platform 2](https://data-artisans.com/blog/da-platform-2-stateful-stream-processing-with-apache-flink-made-easier). The 'Application Manager' is an integral component of the platform and is the central coordination point between commands from the user and orchestration of cluster resources and Flink jobs.
+
+![dA Platform 2 architecture overview](https://data-artisans.com/wp-content/uploads/2017/09/application_manager_opt4@3x.png)
+
+We aim for dA Platform 2 to be generally available in the first quarter of 2018, providing a commercialy backed platform to the Flink community from the team who helped create the open source project.
+
+### Flink Ecosystem Growth
+
+Flink was added to a selection of distributions and integrations during 2017, making it easier for a wider user base to get started with Flink:
+
+-   [Official Docker image](https://hub.docker.com/r/_/flink/)
+-   [Official DC/OS and Mesos support](https://ci.apache.org/projects/flink/flink-docs-release-1.3/setup/mesos.html)
+-   [A Flink connector](https://data-artisans.com/blog/dellemc-launches-open-source-pravega-complete-apache-flink-connector) for [Pravega](http://pravega.io), Dell/EMC's streaming storage system.
+-   Uber announced AthenaX, a streaming SQL platform [powered by Apache Flink](https://data-artisans.com/blog/uber-introduces-open-source-athenax-streaming-sql-platform-apache-flink).
+
+### Feature Timeline in 2017
+
+Just in time for the end of the year, [our 1.4 release](read the full release announcement](<http://flink.apache.org/news/2017/12/12/release-1.4.0.html>) landed in mid-December culminating 5 months of work and the resolution of more than 900 issues. This is the fifth major release in the 1.x.y series.
+
+Here's a selection of major features added to Flink over the course of 2017:
+
+![Flink Release Timeline 2017]({{ site.baseurl }}/img/blog/flink-releases-2017.png)
```

Comment:
@wints This is still WIP, I was struggling to know what to put in the blank months.

Reply:
This is another one where @fhueske might be able to help out. I can take a closer look later in the week, too.